4_extract/src/extract_model_data.R

```diff
@@ -0,0 +1,69 @@
+#' Extract model data from GLM model runs
+#'
+#' @param model_run_tbl df, a dataframe of model configuration values
+#' @param calibration_group chr, Are you extracting results from a `calibrated` or `uncalibrated` model?
+#' @param run_group chr, See `unique(p1_nldas_model_config$filter_col)` for potential values
+#
+
+extra_glm_output <- function(glm_model_tibble) {
```

Comment:
SC: nitpicky, but I'd spell out 'extract' fully in the function name so that it's clear what the function does